--- ptex-2.4.3/src/build/ptex.pc.in.orig	2024-04-30 20:49:17.000000000 +0200
+++ ptex-2.4.3/src/build/ptex.pc.in	2024-06-12 20:10:24.641778377 +0200
@@ -5,10 +5,10 @@
 # Good tutoral for understanding .pc files:
 # https://people.freedesktop.org/~dbn/pkg-config-guide.html
 
-prefix="@CMAKE_INSTALL_PREFIX@"
-exec_prefix="${prefix}"
-libdir="@CMAKE_INSTALL_PKGCONFIG_LIBDIR@"
-includedir="@CMAKE_INSTALL_PKGCONFIG_INCLUDEDIR@"
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=@CMAKE_INSTALL_PKGCONFIG_LIBDIR@
+includedir=@CMAKE_INSTALL_PKGCONFIG_INCLUDEDIR@
 
 Name: @PROJECT_NAME@
 Description: @CMAKE_PROJECT_DESCRIPTION@
@@ -16,6 +16,6 @@ URL: @CMAKE_PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
 Requires: @pc_req_public@
 Requires.private: @pc_req_private@
-Cflags: -I"${includedir}"
-Libs: -L"${libdir}" -l@PTEX_OUTPUT_NAME@
+Cflags: -I${includedir}
+Libs: -L${libdir} -l@PTEX_OUTPUT_NAME@
 
--- ptex-2.4.3/src/build/CMakeLists.txt.orig	2024-04-30 20:49:17.000000000 +0200
+++ ptex-2.4.3/src/build/CMakeLists.txt	2024-06-12 20:10:57.934931346 +0200
@@ -38,4 +38,4 @@ else()
 endif()
 configure_file("${PROJECT_NAME_LOWERCASE}.pc.in" "${PROJECT_NAME_LOWERCASE}.pc" @ONLY)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME_LOWERCASE}.pc"
-    COMPONENT devel DESTINATION share/pkgconfig)
+    COMPONENT devel DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
