--- ptex-2.5.4/CMakeLists.txt.orig	2026-08-20 00:24:34.000000000 +0200
+++ ptex-2.5.4/CMakeLists.txt	2026-09-14 19:43:14.716769264 +0200
@@ -107,11 +107,7 @@ if (MSVC)
 endif()
 
 # Set the default Ptex library to use for tools and tests.
-if (PTEX_BUILD_STATIC_LIBS)
-    set(PTEX_LIBRARY Ptex_static)
-else()
     set(PTEX_LIBRARY Ptex_dynamic)
-endif()
 
 if (${PRMAN_15_COMPATIBLE_PTEX})
     add_definitions(-DPTEX_NO_LARGE_METADATA_BLOCKS)
